spec/features/admin/variants_spec.rb

```diff
@@ -1,30 +1,86 @@
 require 'spec_helper'
 
+# TODO check the filter are passed around as expected
```

Comment:
I dont think we should have TODO entries on the code. If this is something to be done in the future we should create a github issue to represent that task.

Reply:
Removed, I did not mean to commit this.